### DISCUSSION TITLE
Hull water editing in editor only works after saving and reloading

### DISCUSSION THREAD

**Discussion by UnhappyWithMyUsername** [FakeFishGames/Barotrauma#13605] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When trying to fill hulls with water in the submarine editor using the "water" command, newly placed hulls will not be fillable.
Hulls will only become fillable when you save the module/submarine and reload it.

### Reproduction steps

1. Create a hull.
2. Activate the water editing with the "water" command.
3. Try to fill the hull by holding left-click in it. Nothing will happen.
4. Save the hull as a new module/submarine.
5. Load your new module/submarine.
6. Try filling the hull by holding left-click in it again. It will now fill properly.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by PanmanS** [comment]
Cannot replicate on my end.

**Comment by Jasontti** [comment]
Thank you for your report.

Was able to replicate this with the steps provided. Opening a ticket for it https://github.com/FakeFishGames/Barotrauma/issues/16442

**Comment by Regalis11** [comment]
Fix in our private repo now, fix coming up in a future unstable build and update. :)
